samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm

```diff
@@ -42,7 +42,6 @@ - (void)viewDidLoad {
         [self setupAccelerometer];
     } else {
         accLabel = [[UILabel alloc] initWithFrame:CGRectMake(30, accHeight, 300, 50)];
-        [accLabel setBackgroundColor:nil];
```

Comment:
Might want to use [UIColor blackColor] for more clarity

---
In reply to: [64473453](https://github.com/Microsoft/WinObjC/pull/512#discussion_r64473453) [](ancestors = 64473453)